Content.Shared/Damage/Systems/PassiveDamageSystem.cs

```diff
@@ -37,18 +37,42 @@ public override void Update(float frameTime)
             if (comp.NextDamage > curTime)
                 continue;
 
-            if (comp.DamageCap != 0 && damage.TotalDamage >= comp.DamageCap)
+            // Check if there are any passive damage entries.
+            if (comp.PassiveDamageList.Count == 0)
                 continue;
 
+            foreach (var entry in comp.PassiveDamageList)
+            {
+                // If the entity has no damage entry, skip.
+                if (entry.Damage == null)
+                    continue;
+                // If the entity is not in a valid state for this entry, skip.
+                if (entry.AllowedStates == null || !entry.AllowedStates.Contains(mobState.CurrentState))
+                    continue;
+
+                // If the damage cap is disabled, or the total damage is less/equal to the cap, add to the sum.
+                if (entry.DamageCap == 0 || damage.TotalDamage <= entry.DamageCap)
```

Comment:
Instead of doing `if, else if` and adding `comp.DamageSum += entry.Damage` at the end of each branch, it's cleaner to check for failed statements and run `continue;`.